ParabolInc/parabol
# chore(prompt-to-join): add request to join domain mutation

How to test:

- Follow previous PR (#8003) testing guide to trigger prompt to join org notification (or manually add it to database) 
- Check that request button works both for notification and snackbar
- Check that new `DomainJoinRequest` record is created
- Check that no duplicate records are ever created
